docs/tutorials/python/file.md

```diff
@@ -43,6 +43,11 @@ In this tutorial you will:
 
 ## 1. Upload several files to Synapse
 
+!!! warning "Uploading Large Files"
+    If you are uploading very large files (>100 GB each), consider using the **async OOP API with sequential uploads** for better performance and resource management.
+
+    For large file uploads, see the `execute_walk_file_sequential()` function in [uploadBenchmark.py](https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/docs/scripts/uploadBenchmark.py) as a reference implementation. This approach uses `asyncio.run(file.store_async())` with the newer async API, which has been optimized for handling very large files efficiently. In benchmarks, this pattern successfully uploaded 45 files of 100 GB each (4.5 TB total) in approximately 20.6 hours.
```

Comment:
I like that we are diving into this to get better suggestions. However I don't think that anyone outside of us knows what the `async OOP API` means. I would suggest to leave it out and be a bit more literal by pointing to specific methods or scripts.